docs/GetStarted/getting_started_macos_bazel.md

```diff
@@ -0,0 +1,164 @@
+# Getting Started on MacOS with Bazel
+
+<!--
+Notes to those updating this guide:
+
+    * This document should be __simple__ and cover essential items only.
+      Notes for optional components should go in separate files.
+
+    * This document parallels getting_started_linux_bazel.md and
+      getting_started_windows_bazel.md
+      Please keep them in sync.
+-->
+
+This guide walks through building the core compiler and runtime parts of IREE
+from source. Auxiliary components like the Python bindings and Vulkan driver are
+not documented for MacOS at this time.
+
+IREE is not officially supported on MacOS at this time. It may work, but it is
+not a part of our open source CI, and may be intermittently broken.
+Contributions related to MacOS support and documentation are welcome however.
+
+## Prerequisites
+
+### Install Bazel
+
+Install the version of Bazel found in
+[`.bazelversion`](https://github.com/google/iree/blob/master/.bazelversion).
+
+#### Install Xcode.
+
+Running this may be sufficient (and takes up much less disk space):
+
+```
+$ xcode-select --install
+```
+
+If you run into complications, see the
+[official docs](https://docs.bazel.build/versions/master/install-os-x.html#install-with-installer-mac-os-x)
+for guidance.
+
+#### Download and install Bazel from a binary.
+
+```
+# Example using Bazel version `2.1.0`
+$ export BAZEL_VERSION=2.1.0  # Use the version in .bazelversion
+
+# Download and install
+$ curl -LO "https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-darwin-x86_64.sh"
+$ chmod +x "bazel-${BAZEL_VERSION}-installer-darwin-x86_64.sh"
+$ ./bazel-${BAZEL_VERSION}-installer-darwin-x86_64.sh
+
+# Add to path in `.zshrc`. Swap with `.bashrc` if you are using bash.
+$ echo -e '\n# Add bazel to PATH:\nexport PATH="${PATH}:${HOME}/bin"' >> ~/.zshrc
+$ source ~/.zshrc
+
+# Confirm the version
+$ bazel --version # 2.1.0
+```
+
+### Install a Compiler
```

Comment:
Do we need this? I think by default MacOS uses Clang anyway.

Reply:
I included that section to make this doc mirror the linux and windows ones, which have this step (comment on line 9).

Since there really isn't anything to do here now that the environment variables aren't needed I removed this section from both macos_bazel and macos_cmake.